secant/Features/Settings/SettingsView.swift

```diff
@@ -22,7 +22,17 @@ struct SettingsView: View {
                 .primaryButtonStyle
                 .frame(height: 50)
                 .padding(.horizontal, 30)
-                
+
+                Button {
+                    exportAndShareLogs()
+                } label: {
+                    Text("Export & share logs")
+                }
+                .primaryButtonStyle
```

Comment:
isn't indentation wrong here?

Reply:
Not wrong but different syntax, I changed it so it's the same as other buttons in the view.